kotlin-native/Interop/Runtime/src/main/kotlin/kotlinx/cinterop/Utils.kt

```diff
@@ -727,3 +735,43 @@ public fun COpaquePointer.readBytes(count: Int): ByteArray {
     nativeMemUtils.getByteArray(this.reinterpret<ByteVar>().pointed, result, count)
     return result
 }
+
+@ExperimentalForeignApi
+public fun setMemory(mem: NativePointed, value: Byte, size: Long) {
```

Comment:
Could you please add a small KDoc for all new public functions?

Reply:
Will do!